src/stacking-group.tsx

```diff
@@ -0,0 +1,155 @@
+import * as React from 'react';
+import Avatar from './avatar';
+import Image from './image';
+import * as styles from './stacking-group.css';
+import {useIsInverseVariant} from './theme-variant-context';
+import {vars} from './skins/skin-contract.css';
+import classNames from 'classnames';
+
+type dataArrayType = {
+    src?: string;
+    initials?: string;
+};
+
+type StackingGroupProps = {
+    size: 24 | 32 | 64;
+    data: dataArrayType[];
+    type: 'avatar' | 'image';
+    stacked?: boolean;
+};
+
+const StackingGroup: React.FC<StackingGroupProps> = ({size = 24, data, type, stacked = false}) => {
+    // Due to the fact that the image does not have a variant to show the surplus of the array,
+    // as in the case of the avatar, a pseudo-image had to be created in this component
+
+    const isInverse = useIsInverseVariant();
+    const backgroundColor = isInverse ? vars.colors.brandHigh : vars.colors.brandLow;
+    const textColor = isInverse ? vars.colors.textPrimaryInverse : vars.colors.brand;
+
+    let maxItems = 0;
+    if (stacked) {
+        maxItems = 4;
+    } else {
+        maxItems = 3;
+    }
```

Comment:
this shouldn't be harcoded here, it should be a `maxItems` prop and be `Infinity` by default

Reply:
According to history, there is a maximum limit of 5 if the stacked is true and 4 if it is false
<img width="552" alt="Captura de tela 2023-05-08 132759" src="https://user-images.githubusercontent.com/123959276/236878866-90cc5dd5-5ca5-4468-b49d-72a66760cc5b.png">

Reply:
This maximun is just for the Vivo Novo app, but not in terms of mística component. Mística component provide more flexibility because have to be built as a global component. Later, when a product (in this case novo app) want to use this component, can provide its logic for a specific use.